test/data/AccountProof.json

```diff
@@ -0,0 +1,6 @@
+  "stateRoot": "0x58810687b84d5bddc1e9e68b2733caa4a8c6c9e7dd5d0b2f9c28b4bbf5c6f850"
+}
```

Comment:
I am also using the same file with some modifications.
Let's sync up on this. You can take my file. Data point name has also modified.